### ISSUE TITLE
iot describe_thing_type does not include thingTypeArn

### ISSUE THREAD

**Issue by ezramorris** [spulec/moto#3623]
`describe_thing_type` in the iot mock does not return `thingTypeArn`.

This should be included as per the [boto3 documentation](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/iot.html#IoT.Client.describe_thing_type).

To reproduce:

```
>>> import boto3
>>> import moto
>>> mock = moto.mock_iot()
>>> mock.start()
>>> client = boto3.client('iot')
>>> client.create_thing_type(thingTypeName='dummy')
{'ResponseMetadata': {'HTTPStatusCode': 200, 'HTTPHeaders': {'server': 'amazon.com'}, 'RetryAttempts': 0}, 'thingTypeName': 'dummy', 'thingTypeArn': 'arn:aws:iot:eu-west-1:1:thingtype/dummy'}
>>> client.describe_thing_type(thingTypeName='dummy')
{'ResponseMetadata': {'HTTPStatusCode': 200, 'HTTPHeaders': {'server': 'amazon.com'}, 'RetryAttempts': 0}, 'thingTypeName': 'dummy', 'thingTypeId': '4132577a-8127-410d-9bd0-20889b9ad242', 'thingTypeProperties': {}, 'thingTypeMetadata': {'deprecated': False, 'creationDate': datetime.datetime(2021, 1, 28, 10, 53, 3, 18000, tzinfo=tzlocal())}}

```

The issue appears to be that `moto.iot.models.FakeThingType.to_dict` does not include the `thingTypeArn` attribute.

**Comment by bblommers** [comment]
Thanks for raising this @ezramorris - marking it as an enhancement!

**Comment by bblommers** [comment]
This is now fixed as of moto >= 1.3.17.dev232
